tests/entrypoints/openai/test_return_tokens_as_ids.py

```diff
@@ -16,19 +16,28 @@
 from .test_completion import MODEL_NAME
 
 
-@pytest.fixture(scope="module")
-def server_with_return_tokens_as_token_ids_flag(
-        default_server_args):  # noqa: F811
-    args_with_flag = default_server_args + ["--return-tokens-as-token-ids"]
-    with RemoteOpenAIServer(MODEL_NAME, args_with_flag) as remote_server:
-        yield remote_server
+@pytest.fixture(scope="module", params=[True, False])
```

Comment:
```suggestion
@pytest.fixture(scope="module", params=[True, False])
```

From my understanding, there is no need to pass `params` when you're already requesting the fixture with `indirect=True`.

Reply:
You're absolutely correct. Thanks!